### PR TITLE
[Merged by Bors] - chore: forward port leanprover-community/mathlib#18657

### PR DESCRIPTION
**Pull request by Parcly-Taxel**
---

- [x] depends on: leanprover-community/mathlib#18690

[Open in Gitpod](https://gitpod.io/from-referrer/)

* [`order.complete_boolean_algebra`@`c5c7e2760814660967bc27f0de95d190a22297f3`..`e8ac6315bcfcbaf2d19a046719c3b553206dac75`](https://leanprover-community.github.io/mathlib-port-status/file/order/complete_boolean_algebra?range=c5c7e2760814660967bc27f0de95d190a22297f3..e8ac6315bcfcbaf2d19a046719c3b553206dac75)
* [`order.hom.complete_lattice`@`bcfa726826abd57587355b4b5b7e78ad6527b7e4`..`e8ac6315bcfcbaf2d19a046719c3b553206dac75`](https://leanprover-community.github.io/mathlib-port-status/file/order/hom/complete_lattice?range=bcfa726826abd57587355b4b5b7e78ad6527b7e4..e8ac6315bcfcbaf2d19a046719c3b553206dac75)

### CODE REVIEW

File: Mathlib/Order/CompleteBooleanAlgebra.lean [hunk 1]
```diff
@@ -20,7 +20,7 @@ distributive Boolean algebras.
 
 ## Typeclasses
 
-* `Order.Frame`: Frame: A complete lattice whose `⊓` distributes over `⨆`.
+* `Order.Frame`: Frm: A complete lattice whose `⊓` distributes over `⨆`.
```
Comment (semorrison) [inline Mathlib/Order/CompleteBooleanAlgebra.lean:23]: Actually, this was a mistake in the original mathlib3 PR. Just undo this change here.

Comment (semorrison) [reply Mathlib/Order/CompleteBooleanAlgebra.lean:23]: In fact, all these changes were mistakes! You can just revert them, so all this PR does is bumps the SHAs. If you'd like to fix mathlib3 as well that would be nice, but not essential.

Comment (Parcly-Taxel) [reply Mathlib/Order/CompleteBooleanAlgebra.lean]: Will leanprover-community/mathlib#18690 do the job over there?

**Comment by semorrison** [comment]
This PR/issue depends on:
* leanprover-community/mathlib#18690
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

**Comment by semorrison** [comment]
This PR/issue depends on:
* ~~leanprover-community/mathlib#18690~~
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

**Comment by semorrison** [comment]
bors merge
